iotdb/src/test/java/org/apache/iotdb/db/utils/datastructure/ListPublicBAOSLongTest.java

```diff
@@ -0,0 +1,126 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.iotdb.db.utils.datastructure;
+
+import java.io.*;
+import java.lang.Thread;
+import java.util.Random;
+
+import static org.apache.iotdb.db.utils.datastructure.ListPublicBAOSPool.ARRAY_SIZE;
+
+public class ListPublicBAOSLongTest {
+  private static final int consumerNum = 1;
+
+  public static void main(String[] args) throws InterruptedException, FileNotFoundException {
+    System.setOut(new PrintStream(new File("C:\\Users\\10334\\Desktop\\out1.txt")));
```

Comment:
The file path is only for Window OS, change it to a more general path.

Reply:
output message to console now